controllers/extendeddaemonsetreplicaset/controller.go

```diff
@@ -223,11 +223,6 @@ func (r *Reconciler) applyStrategy(logger logr.Logger, daemonset *datadoghqv1alp
 		conditions.UpdateExtendedDaemonSetReplicaSetStatusCondition(strategyParams.NewStatus, now, datadoghqv1alpha1.ConditionTypeActive, corev1.ConditionFalse, "", "", false, false)
 		logger.Info("manage canary deployment")
 		strategyResult, err = strategy.ManageCanaryDeployment(r.client, daemonset, strategyParams)
-	case strategy.ReplicaSetStatusCanaryFailed:
-		logger.Info("manage canary failed deployment")
-		strategyResult = &strategy.Result{
-			NewStatus: strategyParams.NewStatus.DeepCopy(),
-		}
```

Comment:
this was added in https://github.com/DataDog/extendeddaemonset/pull/66 but i don't think this case is ever visited

Reply:
right